index.html

```diff
@@ -27,9 +27,21 @@
                 <li>
                     <a href="#contact">Contact</a></li>
                 <li><img class="nav-pic" src="img/Union.png" alt="humburger-icon"></li>
-            </ul>
+                </ul>
 
         </nav>
+        <div id="menu" class="m-menu-continer">
+            <img class="close" alt="Close Botton" src="img/close.png">
+            <ul class="links-container">
+                    <li class="m-menu-list">
+                        <a class="links link1" href="#portfolio">Portfolio</a></li>
+                    <li class="m-menu-list">
+                        <a class="links link2" href="#about">About</a></li>
+                    <li class="m-menu-list">
+                        <a class="links link3" href="#contact">Contact</a>
+                    </li>
+            </ul>
+        </div>

```

Comment:
- [ OPTIONAL] Be careful with blank lines and indentations, do not add spaces or indentations without a reason. You can add two spaces of indentation for readability and do not use the tab key. for more documentation, you can check the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 🤓👍🏻